docker/build-tools/Dockerfile.centos

```diff
@@ -34,3 +38,9 @@ RUN curl -fsSL --output ${LLVM_RELEASE}.tar.xz https://releases.llvm.org/${LLVM_
     rm ./${LLVM_RELEASE}.tar.xz && \
     echo "/opt/llvm/lib" > /etc/ld.so.conf.d/llvm.conf && \
     ldconfig
+
+ENV PATH=/opt/llvm/bin:/usr/local/google-cloud-sdk/bin:$PATH
+
+# hadolint ignore=DL3003
+RUN cd /tmp && git clone https://gn.googlesource.com/gn && \
```

Comment:
Do you have to build a specific version? master works?

Reply:
Looking into this. @jplevyak says it should be stable but will see if we can find the correct version to pin

Reply:
'gn_version': 'git_revision:5ed3c9cc67b090d5e311e4bd2aba072173e82db9',


This matches the wee8  in proxy-wasm master.   It is in the DEPS in the root of the wee8 tar ball in envoy's repository_locations.bzl BTW